nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.h

```diff
@@ -33,6 +33,14 @@ class GeckoVBufBackend_t: public VBufBackend_t {
 		bool ignoreInteractiveUnlabelledGraphics=false
 	);
 
+	void fillVBufAriaDetails(
+		int docHandle,
+		CComPtr<IAccessible2> pacc,
+		VBufStorage_buffer_t* buffer,
+		VBufStorage_controlFieldNode_t* parentNode,
+		std::wstring& roleAttr
+	);

```

Comment:
If this isn't called by any other class, and doesn't use any member variables or functions, it can doesn't need to be declared in the header. It can be entirely within the `cpp` file.

Reply:
`fillVBufAriaDetails` uses the member function `getRelationId`